### ISSUE TITLE
Use of mixup, label smoothing are used?

### ISSUE THREAD

**Issue by bangawayoo** [liyunsheng13/dcd#5]
Hello,
Thank you for the intriguing work! 

I had a question regarding the use of mixup and label smoothing.
The paper mentions that mixup is used for MobileNetV2 x1.0 and dropout is used for ResNet-50: 
`"For MobileNetV2 ×1.0,Mixup (Zhang et al., 2018a) and label smoothing are further added to avoid overfitting."`

But in the command provided, it seems like mixup is used to train ResNet-50. Was mixup or label smoothing used for any other models?
Additionally, regarding the numbers in the paper, are the above mentioned methods used for the static method? 

Thanks!

**Comment by liyunsheng13** [comment]
When I reproduce the result for ResNet-50, I found adding mixup and label smoothing can sightly improve the performance. So I choose to release the better model. I don't have time to fully ablate the influence given by mixup and label smoothing but based on the limited results I have, I found they are not that important. For other DCD models, I do not use mixup at all since the small models cannot be trained with very strong data augmentation techniques. But label smoothing can be added to ResNet-18 and MobileNetV2 x0.5. Still, label smoothing might improve the performance but I think it should not make a big difference (0.2% at most).
For the static model, the results shown in the paper are just copied from the original paper. I add mixup or label smoothing for a few of them but do not observe a big difference. In order to keep results consistent, I just use the ones from the original paper instead of reporting my own results.
